app/player/player.go

```diff
@@ -384,46 +402,51 @@ func (b *chunkGetter) prefetchToLocalCache(startN int) {
 	chunksLeft := len(b.sdBlob.BlobInfos) - startN - 1 // Last blob is empty
 	if chunksLeft <= 0 {
 		return
-	} else if chunksLeft > 10 {
-		prefetchLen = 10
+	} else if chunksLeft > 3 {
```

Comment:
this should prolly be a const, with some explanation of why the value is 3 or 10 or whatever